aat/main_test.go

```diff
@@ -278,10 +325,18 @@ func connectClientCfg(cfg client.Config) (*client.Client, error) {
 
 	var addr string
 	switch scheme {
-	case "http", "ws", "tcp":
-		addr = fmt.Sprintf("%s://%s/", scheme, tcpAddr)
+	case "http", "https":
```

Comment:
In this `switch`, `"http"` should be handled the same as `"ws"` and `"tcp"`.  `"https"` should be handled the same as `"wss"` and `"tcps"`